.github/actions/summarizer/index.js

```diff
@@ -0,0 +1,106 @@
+const github  = require('@actions/github');
+const core    = require('@actions/core');
+const token   = process.env.GITHUB_TOKEN;
+const octokit = new github.getOctokit(token);
+const { BedrockRuntimeClient, InvokeModelCommand } = require("@aws-sdk/client-bedrock-runtime");
+
+(async () => {
+  const context = github.context;
+  const payload = context.payload;
+  const author = payload.comment.user.login;
+  const authorized = ["OWNER", "MEMBER"].includes(payload.comment.author_association);
+  // Ensure that invoker is either a Owner or member of awslabs / amazon-eks-ami
+  if (!authorized) {
+    console.log(`Comment author is not authorized: ${author}`);
+    return;
+  }
+  console.log(`Comment author is authorized: ${author}`);
+
+  // Split the command into parts
+  const parts = process.env.COMMENT_BODY.trim().split(' ');
```

Comment:
What if the comment has more than just the command in it? what if it has multiple commands?

Reply:
Currently, it requires that you only type a single command. Working on moving the summarizer over to the existing bot.